generator/namespaces.py

```diff
@@ -45,6 +44,24 @@ def _get_views(uri):
     return views
 
 
+def _get_looker_views(
+    app_name: str, variants: dict, db_views: Dict[str, Dict[str, List[List[str]]]]
+):
+    views = defaultdict(list)
```

Comment:
Using a `defaultdict` here is misleading because the default is never used, an error is thrown before a value could be assigned more than once, and the result is immediately cast to a regular `dict`
```suggestion
    views = {}
```

Reply:
I agree, it was a holdover from the previous version of this code.